test/integration/testutils/strings.go

```diff
@@ -45,3 +41,16 @@ func GetLastSplitElement(value string, sep string) string {
 	splitted := strings.Split(value, sep)
 	return splitted[len(splitted)-1]
 }
+
+func EnvNames(branchName string) []string {
+	if branchName == "release-please--branches--main" {
```

Comment:
I like making this a function, as longer term a switch/case can be used when we have different types of examples that use different env names.